loops/task_05.py

```diff
@@ -0,0 +1,9 @@
+str_list = ['apple', 'banana', 'cherry', 'date', 'elderberry']
+print(str_list)
+
+reversed_str_list = []
+for i in range(len(str_list) - 1, -1, -1):  
+    reversed_str_list.append(str_list[i])
```

Comment:
Possible improvement
There are easier option to create reversed sequence)
Also check `list` method `reverse()` and `reversed()` built-in function, compare them

```suggestion
reversed_str_list = str_list[::-1] # [begin:end:step]
```